example-dmn-runtime-service/Dockerfile

```diff
@@ -1,5 +1,3 @@
-FROM quay.io/alfresco/alfresco-dmn-runtime-service:7.1.0.M5
+FROM quay.io/alfresco/alfresco-dmn-runtime-service:develop
```

Comment:
Are we changing this to 7.1.0.M6 after the release?

Reply:
yep

Reply:
or just a tag